getting-started.adoc

```diff
@@ -1,154 +1,192 @@
 [[getting-started-with-raincatcher]]
-Getting Started with Raincatcher
---------------------------------
+= Getting Started with Raincatcher
 
-[[overview]]
-Overview
-~~~~~~~~
+== Overview
+
+Raincatcher is a set of templates and a field workforce management (WFM) application that connects a business' back-office with its fleet of mobilized employees.  Raincatcher leverages mobile hardware, making use of each employee's mobile phone.
+
+Raincatcher consists of a set of reusable modules that can be included in your application using http://browserify.org/[browserify]. 
+
+
+Raincatcher is based on the principle of multiple,
+loosely-coupled modules communicating through the mediator pattern.
+Raincatcher modules make use of the
+https://addyosmani.com/largescalejavascript/[mediator pattern] to enable
+loose coupling between the Raincatcher modules and their consuming
+applications / modules as shown in the following diagram:
+
+image:assets/images/wfm-architecture.png[Raincatcher Architecture]
```

Comment:
@pmdarrow  my problem with the architecture doc, is is that it's a demo architecture. I'll revisit this, I'm already  working on a version of the doc that brings us from the general to specific, currently the doc is very 'demo' oriented, ie very specific